src/provider_definitions.jl

```diff
@@ -1,37 +1,38 @@
-function process(r::JSONRPC.Request{Val{Symbol("textDocument/definition")},TextDocumentPositionParams}, server)
+function process(r::JSONRPC.Request{Val{Symbol("textDocument/definition")}, TextDocumentPositionParams}, server)
     tdpp = r.params
     doc = server.documents[tdpp.textDocument.uri]
-    offset = get_offset(doc, tdpp.position.line+1, tdpp.position.character+1)
-    ns = get_names(tdpp.textDocument.uri, offset, server)
+    offset = get_offset(doc, tdpp.position.line + 1, tdpp.position.character + 1)
     word = get_word(tdpp, server)
+    y, Y, I, O, scope, modules, current_namespace = get_scope(doc, offset, server)
+
+    locations = []
```

Comment:
If these are type `Any` I think they should be enforced for clarity. And if not then there will be perf boost if you type them out i.e. `Int[]` or similar